libcontainer/cgroups/utils.go

```diff
@@ -21,49 +21,31 @@ const cgroupNamePrefix = "name="
 
 // https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt
 func FindCgroupMountpoint(subsystem string) (string, error) {
-	f, err := os.Open("/proc/self/mountinfo")
+	mounts, err := mount.GetMounts()
```

Comment:
Yeah, it's like super-inefficient, 60% of container start was spent in that func.